utils/markdown/renderer.tsx

```diff
@@ -139,11 +137,11 @@ export default class Renderer extends marked.Renderer {
     return out;
   }
 
-  heading(text, level) {
+  heading(text: string, level: number) {
     return `<h${level} class="markdown__heading">${text}</h${level}>`;
   }
 
-  link(href, title, text, isUrl) {
+  link(href: string, title: string, text: string, isUrl: boolean) {
```

Comment:
`link` method signature does not match parent class. What should be the best move here?

Reply:
Not sure, @hmhealey what are your thoughts?